wp-content/themes/core/pcss/components/button/_outline.pcss

```diff
@@ -14,29 +15,46 @@ a.c-btn-outline {
  * Button: Modifier - Inverse
  * ------------------------------------------------------------------------- */
 
-.c-btn--inverse,
-a.c-btn--inverse {
+.c-btn-outline--inverse,
+a.c-btn-outline--inverse,
+a[data-mce-selected].c-btn-outline--inverse {
+	border-color: var(--color-white);
+	color: var(--color-white);
 
-	&.c-btn-outline {
-		border-color: var(--color-white);
+	&:visited {
 		color: var(--color-white);
+	}
+
+	&:hover,
+	&:focus {
+		border-color: var(--color-link-hover);
+		color: var(--color-link-hover);
+	}
+
+	&:active {
+		background-color: var(--color-outer-space);
+		border: 1px solid var(--color-off-white);
+		color: var(--color-white);
+	}
+
+	&[disabled] {
+		background-color: var(--color-outer-space);
+		border: 1px solid var(--color-nevada);
+		color: var(--color-nevada);
+	}
+
+	/* CASE: Button Modifier - Icon */
+	&.c-btn--icon:not([disabled]) {
```

Comment:
Couple of questions:

* Do we need the additional `not` on this?
* Seeing this particular modified sprinkled throughout these button partials. Would it may make sense, given the primary purpose is to add an icon to a button to let it have it's own partial so all of it's styles could be together in one place? Let's further discuss this.